spec/api/client_spec.rb

```diff
@@ -442,6 +443,16 @@
 
       expect(a_request(:delete, url)).to have_been_made.once
     end
+    
+    it "makes a request with a valid URI" do
+      new_tag = "new tag"
+
+      new_tag_url = "#{snitch_url}/#{token}/tags/#{new_tag}" 
+      stub_request(:delete, new_tag_url).to_return(:body => body, :status => 200)
```

Comment:
This stub will never get triggered since `new_tag_url` does includes the space that we needed to escape. I'm going to remove this bit so we can get it merged.